control-plane/stor-port/src/types/v0/transport/pool.rs

```diff
@@ -117,6 +117,7 @@ impl From<CtrlPoolState> for models::PoolState {
 
 rpc_impl_string_id!(PoolId, "ID of a pool");
 rpc_impl_string_uuid!(PoolUuid, "UUID of a pool");
+rpc_impl_string_id!(Labels, "Labels of a pool");
```

Comment:
Why are we adding labels as an identifier?

Reply:
It is used as a filter criteria [here ](https://github.com/sinhaashish/mayastor-control-plane/blob/d4608a04204c404ddd8733ee0c5eec667a3896c6/control-plane/stor-port/src/types/v0/transport/misc.rs#L51)